x-pack/test_serverless/api_integration/test_suites/security/cloud_security_posture/helper.ts

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { SuperTest, Test } from 'supertest';
```

Comment:
It would be good to reuse the helpers and mock_data but importing from `tests` directly is breaking `no_boundary_crossing` check of Kibana. If we decide to do that we need to move our test helpers to a shared package (or find another possible way)

Reply:
I saw many teams have their own packages, we might need to create our own as well

Reply:
as discussed long term it makes sense. For now I decided to go with importing from `tests/` and suppressing no boundary crossing error